pkg/storaged/content-views.jsx

```diff
@@ -73,7 +73,7 @@ function next_default_logical_volume_name(client, vgroup, prefix) {
     return name;
 }
 
-export function set_crypto_options(block, readonly, auto) {
+export function set_crypto_options(block, readonly, auto, nofail, netdev) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)